library/nameops.mli

```diff
@@ -50,16 +50,57 @@ val increment_subscript : Id.t -> Id.t
 
 val forget_subscript    : Id.t -> Id.t
 
+module Name : sig
+
+  include module type of struct include Names.Name end
+
+  val fold_right : (Id.t -> 'a -> 'a) -> Name.t -> 'a -> 'a
+  (** [fold_left f a x] is [f a id] if [x] is [Name id], and [a] otherwise. *)
```

Comment:
Same typo as above. By the way, wouldn't be easier to maintain those comments if they were only in the `mli` file?